pyra/hardware.py

```diff
@@ -0,0 +1,422 @@
+"""
+..
+   hardware.py
+
+Functions related to the dashboard viewer.
+"""
+# standard imports
+import json
+
+# lib imports
+import GPUtil
+import plotly
+import psutil
+
+# local imports
+from pyra import definitions
+from pyra import helpers
+from pyra import locales
+from pyra import logger
+
+_ = locales.get_text()
```

Comment:
Why is the result of locales.get_text() an underscore, since you clearly care about the result?

Reply:
It makes it easier to use. Basically you just put `_("Hello World!")` and then it will look for the translation in the language defined. That's also the command that Babel looks for to extract strings from by default.